kafka-ui-react-app/src/components/Topics/List/ListHeader.tsx

```diff
@@ -0,0 +1,40 @@
+import { TopicColumnsToSort } from 'generated-sources';
```

Comment:
I like the approach with order-able table cell component. But in this case I would suggest
- create common component for order-able table cell ´common/table/cell´
- use cell component directly without header wrapper